sources/csv/csv.go

```diff
@@ -18,6 +19,9 @@ func (t *table) start(reader *csv.Reader) {
 	defer close(t.rows)
 
 	headers, err := reader.Read()
+	if _, ok := err.(*csv.ParseError); ok {
+		log.Printf("csv.go received encoding/csv.ErrQuote, this can happen when a csv uses the wrong delimiter. Current delimiter is %q.", reader.Comma)
```

Comment:
Also, instead of being a print statement, this should be just a different error message. So something like:

```go
if err != nil {
  if _, ok := err.(*csv.ParseError); ok {
    err = [new error object]
  }
  t.handleErr(err)
  return
}
```